### PR TITLE
Add rusha@0.8.5 w/ npm auto-update

### PR DESCRIPTION
**Pull request by hare1039**
Pull request for issue: #10565
Related issue(s): # #

@cdnjs/intern3 Please help me review on this PR. Thanks.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/srijs/rusha 
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/rusha
 * License and its reference: MIT, [ref](https://github.com/srijs/rusha/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 5
   - Count of stars: 193
   - Count of forks: 27
 * NPM download stats (optional):
   - Downloads in the last day: 4,601
   - Downloads in the last week: 30,956
   - Downloads in the last month: 109,243

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by hare1039** [comment]
@pvnr0082t Please help me review on this PR. Thanks.

File: ajax/libs/rusha/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "url": "https://github.com/srijs/rusha"
+  },
+  "author": "Sam Rijs",
+  "license": "MIT",
+  "npmName": "rusha",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "rusha!(*.sweet).js"
```
Comment (pvnr0082t) [inline ajax/libs/rusha/package.json]: @hare1039 We can just use `"rusha.!(sweet*)"`

Comment (hare1039) [inline ajax/libs/rusha/package.json]: @pvnr0082t I fix the rule. Please review this PR again. Thanks.

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

**Pull request merged**